examples/tree_1d_dgsem/elixir_shallowwater_parabolic_bowl.jl

```diff
@@ -0,0 +1,109 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations1D(gravity_constant=9.81)
+cfl = 0.3
+
+
+function parabolic_bowl_analytic_1D_H(gravity, x,t)
+  a = 1
+  h_0 = 0.1
+  σ = 0.5
+  ω = sqrt(2*gravity*h_0)/a
+
+  H = σ * h_0/a^2 * (2*x[1]*cos(ω*t) - σ) + h_0
+  return H
+end
+
+# Implemented based on Wintermeyer (here seen in 2D)
+function initial_condition_parabolic_bowl(x, t, equations:: ShallowWaterEquations1D)
+  a = 1
+  h_0 = 0.1
+  σ = 0.5
+  ω = sqrt(2*equations.gravity*h_0)/a
+
+  v = -σ*ω*sin(ω*t)
+
+  b = h_0 * x[1]^2/a^2
+
+  H = max(b, parabolic_bowl_analytic_1D_H(equations.gravity, x, t))
+
+  H = max(H, b + equations.threshold_limiter)
```

Comment:
I would add a one line comment here as to what the `threshold_limiter` is doing and its value. You are taking the default value here from the shallow water constructor.

I know that this value is documented in `shallow_water_1d.jl` but it is good to put a comment here as well. One cannot assume that a user will read the source code.